orchestrator/relayer/batch_relaying.go

```diff
@@ -1,89 +1 @@
 package relayer
-
-import (
-	"context"
-
-	"github.com/ethereum/go-ethereum/common"
-	"github.com/pkg/errors"
-	"github.com/umee-network/umee/x/peggy/types"
-)
-
-// RelayBatches checks the last validator set on Ethereum, if it's lower than our latest valida
-// set then we should package and submit the update as an Ethereum transaction
-func (s *peggyRelayer) RelayBatches(ctx context.Context) error {
```

Comment:
This is probably what we want to replace with code to relay message tuple roots.

Reply:
Yeah, I but I feel this PR was already way, way too large 😬 

There are a few things that need to be done:
 - [ ] integration tests need to be cleaned up
 - [ ] message tuple roots creating and add in that method here (above) and in the corresponding interface
 - [ ] ValeSetRootHash needs to be created (note that the validators here used do additionally sign the valsetUpdates with a different signing key from the tendermint one as far as I understand; just noting that as it might be easier to keep it that way instead of changing everything to ecdsa in tendermint directly)
 - [ ] ...